templates/drawer_layout/app/lib/ui.drawer.js

```diff
@@ -0,0 +1,78 @@
+var drawerWindow = function(args) {
+	this.args = args;
+	this.navView = null;
+	this.win = Ti.UI.createWindow(args);
+	
+	if (!this.args.leftWidth) {
+		this.args.leftWidth = 300;
+	}
+	if (!this.args.animationDuration) {
+		this.args.animationDuration = 500;
+	}
+	
+	var context = this;
+	var buttonImg = Ti.UI.createImageView({
+		image: '/images/menu.png'
+	});
+	
+	buttonImg.addEventListener('click', function() {
+		context.menuClickCb && context.menuClickCb();
+	});
+	
+	this.win.leftNavButton = buttonImg;
+};
+
+drawerWindow.prototype.toggleLeft = function() {
+	this.showDrawer();
+};
+
+drawerWindow.prototype.add = function(view) {
+	 if (view.role === 'centerView'){
+	 	this.win.add(Alloy.createController(view.src, view).getView());
+	 }
+	 if (view.role === 'leftView') {
+	 	this.navView = view;
+	 	this.navView.width = this.args.leftWidth;
+	 }
+};
+
+drawerWindow.prototype.open = function(opts) {
+	this.win.open(opts);
+};
+
+drawerWindow.prototype.getView = function() {
+	return this.win;
+};
+
+drawerWindow.prototype.addEventListener = function(event, cb) {
+	this.menuClickCb = cb;
+};
+
+drawerWindow.prototype.showDrawer = function() {
+	var navWin = Alloy.createController(this.navView.src,this.navView).getView();
```

Comment:
Missing space as well. I will not comment on more style issues, as they will be resolved by eslint as well.